openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py

```diff
@@ -31,6 +32,30 @@ def __init__(self):
     def __str__(self):
         return 'mit_hz'
 
+    def _execute_post(self, url, data=None, headers=None, json_data=None):
+        """
+        Execute post request and save the enrollment response.
+        """
+        user_subscription = {}
+        response = requests.post(
+            url=url,
+            data=data,
+            headers=headers,
+            json=json_data,
+        )
+
+        if response.ok:
+            user_subscription = response.json().get('user', {})
+
+        ExternalEnrollment.objects.update_or_create(  # pylint: disable=no-member
+            controller_name=str(self),
+            course_shell_id=json_data.get('course_id'),
+            email=json_data.get('user_email'),
+            defaults={'meta': user_subscription},
+        )
+
+        return response
```

Comment:
Yes, because its parent class uses it.
Ref: https://github.com/Pearson-Advance/openedx-external-enrollments/blob/master/openedx_external_enrollments/external_enrollments/base_external_enrollment.py#L45